### PR TITLE
Images and videos in Feed Cards

### LINKED ISSUE #407: Cards not showing the images and videos
* Due to the state mutation happening in `feed-card`, so that part of the code was removed.

* Make sure that it works again.

### PR DESCRIPTION
**Pull request by hemantjadon**
Images and video urls which were removed during the restructuring of application are implemented back.

**Link to live demo (if appropriate):https://hemantjadon.github.io/loklak_search** 

**Closes #407**

### CODE REVIEW

File: src/app/feed/feed-card/feed-card.component.ts [hunk 1]
```diff
@@ -131,31 +135,32 @@ export class FeedCardComponent implements OnInit {
 		}
 	}
 
-	private sanitizeAndEmbedURLs(links) {
-		links.forEach((link, i) => {
-			const videoid = links[i].match(/(?:https?:\/{2})?(?:w{3}\.)?youtu(?:be)?\.(?:com|be)(?:\/watch\?v=|\/)([^\s&]+)/);
+	private sanitizeVideoURLs() {
+		this.feedItem.videos.forEach((video) => {
+			const videoid = video.match(/(?:https?:\/{2})?(?:w{3}\.)?youtu(?:be)?\.(?:com|be)(?:\/watch\?v=|\/)([^\s&]+)/);
```
Comment (vibhcool) [inline src/app/feed/feed-card/feed-card.component.ts]: you can also use twitter videos that are extracted. see this -> 
http://loklak.org/api/search.json?q=AnyQuery&filter=video
```

    {
.
.
.
.
.
    "videos": [
        "https://video.twimg.com/ext_tw_video/877218026457894913/pu/vid/480x480/AGYBvF98jPR42vB_.mp4",
        "https://video.twimg.com/ext_tw_video/877218026457894913/pu/pl/LHfROVYC5QpWJNwI.m3u8",
        "https://video.twimg.com/ext_tw_video/877218026457894913/pu/vid/720x720/_WQJWOUfya7eMfe2.mp4",
        "https://video.twimg.com/ext_tw_video/877218026457894913/pu/vid/240x240/NLGKdm_ubGAY9uWv.mp4"
      ],
      "videos_count": 4,
      "mentions": [],
      .
.
.
.}
```

Comment (vibhcool) [inline src/app/feed/feed-card/feed-card.component.ts]: here there are different quality videos available (see in the url) like 480*480, 720*720, 240*240

Comment (hemantjadon) [inline src/app/feed/feed-card/feed-card.component.ts]: @vibhcool Yes I will look into it, but as of now whatever media media is coming from the API is of youtube. Am I wrong here ?

Comment (vibhcool) [inline src/app/feed/feed-card/feed-card.component.ts]: @hemantjadon , media is also coming from twitter embedded videos. try queries like http://loklak.org/api/search.json?q=video&filter=video&count=100&source=twitter

Comment (hemantjadon) [inline src/app/feed/feed-card/feed-card.component.ts]: Yes, thanks @vibhcool for pointing this out, though the videos are mostly from youtube, there are other providers as well. I will look into that, for now I just want to make the videos to work again in the application. I have created the issue [#415](https://github.com/fossasia/loklak_sear)h/issues/415) for the same and will look to make this better. :)

**Comment by mariobehling** [comment]
Good job!

**Pull request merged**